content/english/pygame-pong/activity-7.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Activity 7 - Define key combination to exit the game."
+date: 2019-07-29T13:24:17-07:00
+weight: 9
+draft: false
+---
+
+# Activity 7
+Time for final activity in this Tutorial.. Yay! You have done an awesome job to be here. 
+
+In this activity you need to change the way game exits anytime by pressing a given key combination. 
+
+Find `// TODO (ACTIVITY 7)` in the code. Pygame library gives you a handy way by using `event` to detect what keystrokes are being pressed anytime during the life of program. 
+Currently the game exits by pressing `'BACKSPACE'` key. You should change it to exit by pressing `'ESC'` key on  keyboard. 
+
+{{% notice tip %}}
+
+If the value of `'BACKSPACE`' key in Pygame library is `K_BACKSPACE` then can you guess the value of `'ESC` key? Give it few tries before you ask for help. 
+Confirm the result after pressing <b>run</b>
+
+{{% /notice %}}
+
+
+## Takeaway 
+You have just changed the way game should exit at any time while running.
```

Comment:
At the end, have something to congratulate the student on finishing the curriculum. 

If you have time, think of some ideas as to what extra things the student can do to expand or change the functionality of the game, and add them here.